README.md

```diff
@@ -5,6 +5,10 @@
 [![Test Coverage](https://api.codeclimate.com/v1/badges/316be6fc3d163dae0cab/test_coverage)](https://codeclimate.com/github/varvet/DesignableKit/test_coverage)
 [![Maintainability](https://api.codeclimate.com/v1/badges/316be6fc3d163dae0cab/maintainability)](https://codeclimate.com/github/varvet/DesignableKit/maintainability)
 
+A collection of class that you can use to give you more options in how to style your buttons and views in interface builder. 
```

Comment:
I would prefer something like "A collection of classes that gives you more options in how to style your buttons and views in the interface builder"